src/widgets/_tooltips.scss

```
@@ -0,0 +1,20 @@
+tooltip.background,
+.overlay-bar {
+    background-color: #{'alpha (#222, 0.9)'};
```

Any way to avoid hardcoding a hex value here?

---

At the very least, you can do this so it's compiled; it doesn't need to be runtime alpha.

```suggestion
    background-color:  rgba(#222, 0.9);
```